src/installing-workflow/configuring-object-storage.md

```diff
@@ -89,6 +90,20 @@ The slugrunner reads credentials from the below locations on the filesystem.
 - Key: `/var/run/secrets/object/store/access-key-id`
 - Secret: `/var/run/secrets/object/store/access-key-secret`
 
+## [deis/controller](https://github.com/deis/controller)
+
+When the controller needs to launch a new buildpack application or scale one up, it uses a [replication controller](http://kubernetes.io/docs/user-guide/replication-controller/). Since the slugrunner needs to download the slug to run, it needs the object storage location of the slug and the object storage credentials.
```

Comment:
"needs to launch or scale a new buildpack application"? `scale one up` reads awkwardly.